scripts/data-utils.js

```diff
@@ -0,0 +1,715 @@
+const stelace = require('./admin-sdk')
+const { get, keyBy, mapValues, pick, isBoolean, unset, isEqual, isUndefined, omitBy, isNil } = require('lodash')
+const pMap = require('p-map')
+const pProps = require('p-props')
+const path = require('path')
+const fs = require('fs')
+const { execSync } = require('child_process')
+const { importCategories } = require('./import-categories')
+
+const log = console.log
+
+const objectTypes = [
+  'assets',
+  'assetTypes',
+  'categories',
+  'customAttributes',
+  'messages',
+  'ratings',
+  'tasks',
+  'transactions',
+  'users',
+  'workflows',
+]
+
+const idTypes = objectTypes.concat([
+  'conversations',
+  'userMainOrganization',
+])
+
+const actions = [
+  'create',
+  'remove',
+  'remove-and-create',
+  'sync',
+  'none',
+]
+
+const initDataScript = 'initDataScript'
+
+class DataManager {
+  constructor () {
+    // data fetched via Stelace API
+    this.existingData = {} // { objectType: [objects] } // except for the objectType 'config'
+
+    // data keyed by alias that was used as reference ID in seed file
+    this.referencedData = {} // { objectType: { alias: object } }
+
+    // created or updated data are stored in this object
+    this.syncedData = {} // { objectType: { alias: object } }
+
+    // data loaded from seed file
+    this.data = {} // { objectType: { alias: payload } }
+
+    this.shouldOnlyRemoveScriptObjects = true
+    this.shouldUpdateConfig = true
+    this.objectsAction = {
+      default: 'remove-and-create'
+    }
+  }
+
+  /**
+   * @param {Object}  [options]
+   * @param {Boolean} [options.shouldOnlyRemoveScriptObjects] - only remove objects created by this script
```

Comment:
Default value seems to be true.

Reply:
Indeed, thanks. I'll update the comment doc.